packages/server/src/index.ts

```diff
@@ -322,12 +328,15 @@ setInterval(() => {
   Object.entries(controlConnections).forEach(([, connection]) => {
     const origin = connection?.origin || 'Unknown';
     connectedDevices += 1;
+    const { memMitm, memFree, memStart } = connection.lastMemory;
 
-    const validMemFree = Number.isFinite(connection.lastMemory.memFree) ? connection.lastMemory.memFree : 0;
+    const validMemFree = valueOrZero(memFree);
```

Comment:
That's a good idea, but one of you will have to prepare for the conflict with #22

Reply:
Yeah I am prepared, fine to merge the other PR :)